README.md

```diff
@@ -25,3 +25,33 @@ cd numba-benchmark
 ```bash
 asv run
 ```
+
+## Comparing two commits
+
+Run `asv` on the first commit:
+
+```bash
+asv run "-1 abcdefg
+```
+
+Run `asv`  on the second commit:
+
+```bash
+asv run "-1 1235567
```

Comment:
```suggestion
asv run "-1 1235567"
```

Reply:
fixed in 5c870d33b7649e3bade1a1fe51c55f529a352c8d